R/dm_get_tables.R

```diff
@@ -0,0 +1,31 @@
+#' Get tables
+#'
+#' `dm_get_tables()` returns a named list of \pkg{dplyr} [tbl] objects
+#' of a `dm` object.
+#'
+#' @inheritParams rlang::args_dots_empty
+#' @param keyed `r lifecycle::badge("experimental")`
+#'   Set to `TRUE` to return objects of the internal class `"dm_keyed_tbl"``
+#'   that will contain information on primary and foreign keys
```

Comment:
be clearer about _how_ this info is contained?

Reply:
Perhaps in the article?

Reply:
yes, and how to retrieve the info